### PR TITLE
MRG: Spelling

### PR DESCRIPTION
**Pull request by Eric89GXL**
Ran `make codespell`, found some errors.

Ready for review/merge.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
I think this is something we could potentially add to `Travis`, should we @agramfort?

File: mne/io/reference.py [hunk 1]
```diff
@@ -274,7 +274,7 @@ def set_bipolar_reference(inst, anode, cathode, ch_name=None, ch_info=None,
     channels will be dropped.
 
     Multiple anodes and cathodes can be specified, in which case multiple
-    vitual channels will be created. The 1st anode will be substracted from the
+    vitual channels will be created. The 1st anode will be subtracted from the
```
Comment (jaeilepp) [inline mne/io/reference.py]: vitual -> virtual

**Comment by Eric89GXL** [comment]
Comments addressed and Travis runs `codespell` now, see `codespell`-based failing version from previous run:

https://travis-ci.org/mne-tools/mne-python/jobs/124778352#L2821

Travis should come back happy since I fixed that error. Ready for review from my end.

**Comment by agramfort** [comment]
thanks @Eric89GXL

**Comment by agramfort** [comment]
maybe try to add this to travis... and see how annoying this is

**Comment by agramfort** [comment]
the already annoying thing is that I need a python3 command on my system...

**Comment by Eric89GXL** [comment]
> maybe try to add this to travis... and see how annoying this is

This PR actually already added it to Travis. I didn't need to add Python3 to get it to work, presumably because Ubuntu ships with a Python3 interpreter.

**Pull request merged**